doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-rest-api-con.adoc

```diff
@@ -0,0 +1,108 @@
+[id='controller-rest-api-con_{context}']
+= {CONTROLLER} REST API for {KIE_SERVER} templates and instances
+
+{PRODUCT} provides a {CONTROLLER} REST API that you can use to interact with your {KIE_SERVER} templates (configurations), {KIE_SERVER} instances (remote servers), and associated KIE containers (deployment units) in {PRODUCT} without using {CENTRAL}. This API support enables you to maintain your {PRODUCT} resources more efficiently and optimize your integration and development with {PRODUCT}.
+
+With the {CONTROLLER} REST API, you can perform the following actions:
+
+* Retrieve information about {KIE_SERVER} templates, instances, and associated KIE containers
+* Update, start, or stop KIE containers associated with {KIE_SERVER} templates and instances
+* Create, update, or delete {KIE_SERVER} templates
+* Create, update, or delete {KIE_SERVER} instances
+
+Requests to the {CONTROLLER} REST API require the following components:
+
+Authentication::
+The {CONTROLLER} REST API requires basic HTTP Authentication or token-based authentication for the following user roles, depending on controller type:
++
+--
+* `rest-all` user role if you installed {CENTRAL} and you want to use the built-in {CONTROLLER}
```

Comment:
I can't find any reference that this role _rest-all_ still works in the workbench. It seems that as long as the user is authenticated with any of the valid roles it should work:
- admin
- analyst
- developer
- user
- manager
- process-admin

@ederign @porcelli can you confirm that as well?

Reply:
@cristianonicolai Oh, okay. That's strange because I have experienced not being able to use the controller in Central until I configure my user with rest-all access. Once I do that, full access to controller and to the Knowledge Store REST APIs for Central. Either way, if you don't see it in the code, then it sounds like a bug.

Reply:
@cristianonicolai I could find a few references to `rest-all` role in our codebase. I'd leave it for now, and maybe create a follow-up JIRA to remove all references to it first, and them update docs.